docker/Dockerfile

```
@@ -8,7 +8,7 @@ ENV ANSIBLE_STDOUT_CALLBACK=debug
 RUN apk add --no-cache --update bash git mysql-client openssh-client postgresql rsync sshpass
 RUN apk add --no-cache --virtual build-dependencies gcc libffi-dev musl-dev
 
-RUN pip install --no-cache-dir ansible
+RUN pip install --no-cache-dir ansible ansible-core==2.17
```

Je propose d'autoriser les MAJ des versions "patch" de `ansible-core` : 
```suggestion
RUN pip install --no-cache-dir ansible ansible-core~=2.17.1
```

Voir [Version specifiers - Python Packaging User Guide](https://packaging.python.org/en/latest/specifications/version-specifiers/#id5) et [Compatible release](https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release)

---

tu as raison @lucasmirloup , ansible-core ne suit pas le semver https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html